net-libs/libtorrent/libtorrent-0.15.5.ebuild

```diff
@@ -59,6 +59,11 @@ src_configure() {
 		disable_instrumentation="--disable-instrumentation"
 	fi
 
+	# scripts/common.m4 calls $GREP at configure time.
```

Comment:
If it's relying on grep, can't we just patch in the search for grep explicitly then?